plasTeX/Imagers/pdf2svg.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+
+import subprocess
+import re
+import xml.etree.ElementTree as ET
+from plasTeX.Imagers import VectorImager
+
+length_re = re.compile('([0-9\\.]*)(.*)')
+class PDFSVG(VectorImager):
+    """ Imager that uses pdf2svg """
+    fileExtension = '.svg'
+    verifications = ['pdflatex --help', 'which pdf2svg', 'pdfcrop --help']
+    compiler = 'pdflatex'
+
+    def executeConverter(self, output):
+        open('images-uncropped.pdf', 'wb').write(output)
+        subprocess.call(["pdfcrop", "images-uncropped.pdf", "images.pdf"], stdout=subprocess.DEVNULL)
+
+        rc = 0
+        page = 1
+        while 1:
```

Comment:
I don't understand why this is a `while` loop. Isn't it iterating over `self.images`? Or at least `range(1, len(self.images)+1)` since you seem to use only the page number.

Reply:
Yeah I think we should do that instead... I really shouldn't blindly copy existing code.